manifests/rhoai/shared/apps/kustomization.yaml

```diff
@@ -8,3 +8,4 @@ resources:
   - ./pachyderm
   - ./watson-x
   - ./rhoai
+  - ./nvidia-nim
```

Comment:
I'm trying not to block your PR, but I have other things I need to do before I can provide the suggestion I was after.

Can we instead, just remove this line from this file. That way it doesn't auto deploy with RHOAI if we don't get it done by 2.12 code freeze?

What I want to do is set you folks up with an overlay that will install your item that you can plug into the DSC resource the Operator deploys -- that way your test env can be "enabled" but the default won't be. I just need to test it out before I can share it.

Reply:
Sure, I have reverted the changes to the kustomization files.